drivers/python/age/age.py

```diff
@@ -39,9 +39,13 @@ def load(self, data: bytes | bytearray | memoryview) -> Any | None:
         return parseAgeValue(data.decode('utf-8'))
 
 
-def setUpAge(conn:psycopg.connection, graphName:str):
+def setUpAge(conn:psycopg.connection, graphName:str, superuser:bool=True):
     with conn.cursor() as cursor:
-        cursor.execute("LOAD 'age';")
+        if superuser:
+            cursor.execute("LOAD 'age';")
+        else:
+            cursor.execute("LOAD '$libdir/plugins/age';")

```

Comment:
This assumes that the shared object is present in the `$libdir/plugins` directory. That may not be correct if DBA does not install age there.

Reply:
I'm under the impression that while $libdir may change on the system, the "plugins" directory is somewhat static and should only contain plugins that may be used by non-super users. So if a DBA installs this in a different directory, it won't work either way. However I'm not a DBA and not fully aware of all postgres specifics. Perhaps we could leave it like this for now and see if/when this requires further configuration options.

Reply:
@daanwa In that case, I suggest we rename the parameter to something more descriptive, such as `in_plugins_dir` or `load_from_plugins`. Additionally, we should explain what this parameter does in the documentation.

Reply:
Yes, agree, that's a better way to go about it. I'll use the `load_from_plugins` name.